Classes/DataProcessing/FormListProcessor.php

```
@@ -0,0 +1,38 @@
+{
+    protected ElementSettings $settings;
+
+    protected FormcycleService $formcycleService;
+
+    public function __construct(private readonly FormcycleServiceFactory $formcycleServiceFactory)
+    {
+    }
+
+    public function process(
+        ContentObjectRenderer $cObj,
+        array $contentObjectConfiguration,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$contentObjectConfiguration'. (undefined)

(UnusedFormalParameter)